src/services/patternMatcher.ts

```diff
@@ -0,0 +1,810 @@
+module ts {
+    // Note(cyrusn): this enum is ordered from strongest match type to weakest match type.

```

Comment:
So that i can search for notes/todos from me.  It's also useful to know who owns notes/todos so you can sweep through your codebase addressing them.  Otherwise, you get todos/notes that proliferate and never get handled.